### PR TITLE
ports tgui list but I made it better

### PR DESCRIPTION
**Pull request by jupyterkat**
## About The Pull Request

I will modularize the modularizable parts soon

tgstation/tgstation#56065

## Why It's Good For The Game

yes

## Changelog
:cl:
add: Tgui input lists
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/styles/main.scss [hunk 1]
```diff
@@ -39,6 +39,7 @@
 @include meta.load-css('./components/Tooltip.scss');
 
 // Interfaces
+@include meta.load-css('./interfaces/ListInput.scss');
```
Comment (MCterra10) [inline tgui/packages/tgui/styles/main.scss:42]:
```suggestion
// Interfaces
// austation -- adds tgui lists
@include meta.load-css('./interfaces/ListInput.scss');
```

**Review by MCterra10 — CHANGES_REQUESTED** [review]

**Comment by MCterra10** [comment]
other than that, all just simple one liner changes. should be fine to merge.

**Comment by celotajstg** [comment]
What makes it better than the tg implementation?

**Comment by jupyterkat** [comment]
> What makes it better than the tg implementation?

go figure

**Review by MCterra10 — APPROVED** [review]

**Comment by MCterra10** [comment]
resolve conflicts I'll actually merge it this time

**Pull request merged**